stdlib/3/multiprocessing/managers.pyi

```diff
@@ -2,9 +2,33 @@
 
 # NOTE: These are incomplete!
 
-from typing import Any
+import queue
+import threading
+from typing import Any, Dict, List, Mapping, Sequence, TypeVar
+
+_T = TypeVar('_T')
+_KT = TypeVar('_KT')
+_VT = TypeVar('_VT')
+
+class Namespace: ...
+
+_Namespace = Namespace
 
 class BaseManager:
     def register(self, typeid: str, callable: Any = ...) -> None: ...
 
+class SyncManager(BaseManager):
+    def BoundedSemaphore(self, value: Any = ...) -> threading.BoundedSemaphore: ...
+    def Condition(self, lock: Any = ...) -> threading.Condition: ...
+    def Event(self) -> threading.Event: ...
+    def Lock(self) -> threading.Lock: ...
+    def Namespace(self) -> _Namespace: ...
+    def Queue(self, maxsize: int = ...) -> queue.Queue: ...
+    def RLock(self) -> threading.RLock: ...
```

Comment:
This actually isn't exactly right, it returns an `AcquirerProxy` of some sort. The other methods here also return proxies of some sort, and that's documented at https://docs.python.org/3/library/multiprocessing.html#multiprocessing.managers.SyncManager.

It's documented that these proxies mostly behave like the objects they proxy, so I think it's probably fine to just pretend they return the objects directly.